content/get-started/exploring-projects-on-github/contributing-to-a-project.md

```diff
@@ -28,11 +28,13 @@ You can contribute by submitting pull requests from your fork to the upstream re
 
 This tutorial uses [the Spoon-Knife project](https://github.com/octocat/Spoon-Knife), a test repository that's hosted on {% data variables.product.prodname_dotcom_the_website %} that lets you test the fork and pull request workflow.
 
+### Creating a new fork
+
 1. Navigate to the `Spoon-Knife` project at https://github.com/octocat/Spoon-Knife.
 1. In the top-right corner of the page, click **Fork**.
 
    ![Screenshot of the main page of repository. A button, labeled with a fork icon and "Fork 59.3k," is outlined in dark orange.](/assets/images/help/repository/fork-button.png)
-1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
+1. Under "Owner," select the dropdown menu and click an owner for the forked repository. (If your name is greyed out, it's because you have forked this project in the past; see "Updating an existing fork" below.)
```

Comment:
```suggestion
1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
   >[!NOTE] If your username is grayed out, it's because the fork already exists. Instead, you should bring your existing fork up to date. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-web-ui)".
```

Reply:
This seems like valuable guidance for beginners 💛 Let's put it in a note form, and link out to the supporting article ✨

Reply:
I agree, this seems very sensible. Let's do it the way you suggest.